Source/Core/DolphinQt/TAS/TASInputWindow.h

```diff
@@ -40,12 +40,16 @@ class TASInputWindow : public QDialog
   void GetSpinBoxU8(QSpinBox* spin, u8& controller_value);
   void GetSpinBoxU16(QSpinBox* spin, u16& controller_value);
 
+  void focusOutEvent(QFocusEvent* event) override;
+  void focusInEvent(QFocusEvent* event) override;
+
```

Comment:
Virtual functions should go above regular methods in definitions. (These don't have to go above the public methods, however.)